src/Uno.Sdk/Sdk/Sdk.props

```diff
@@ -71,6 +74,10 @@ Copyright (C) Uno Platform Inc. All rights reserved.
 		</When>
 	</Choose>
 
+	<ItemGroup>
+		<UpToDateCheckInput Include="**\*.xaml" Exclude="bin\**\*.xaml;obj\**\*.xaml" />
+	</ItemGroup>
```

Comment:
I wonder if it should be Uno only. It may not cause issues to WinUI, but it may be duplicates.